frameworks/person-escort-record/questions/indication-of-self-harm-or-suicide.yml

```diff
@@ -15,6 +15,10 @@ options:
   -
     label: "Yes"
     value: "Yes"
+    flags:
```

Comment:
We may want to move these flags to the questions on the next step, otherwise I don't think any detail will be pulled into the alert summary.